src/components/ImageCard/Hotspot.jsx

```diff
@@ -128,7 +134,11 @@ const Hotspot = ({
         triggerId={`hotspot-${x}-${y}`}
         tooltipId={`hotspot-${x}-${y}`}
       >
-        {content}
+        {React.isValidElement(content) ? (
+          content
+        ) : (
+          <HotspotContent {...content} locale={locale} renderIconByName={renderIconByName} />
+        )}
```

Comment:
Could we promote composition with `children` instead of passing elements as props? This will eliminate the need to prop drill `locale`

```suggestion
        {content ? content : children}
```

Also add a deprecation warning for `content` so we can remove the conditional at the next major release.